frontend/src/components/AddBotpack.svelte

```diff
@@ -0,0 +1,208 @@
+<script lang="ts">
+import toast from "svelte-5-french-toast";
+import { App } from "../../bindings/gui";
+import Modal from "./Modal.svelte";
+import ProgressBar from "./ProgressBar.svelte";
+
+const OFFICIAL_BOTPACK_REPOS = [
+  "VirxEC/botpack-test",
+  "VirxEC/pytorch-archive",
+];
+
+let {
+  parentVisible = $bindable(false),
+  visible = $bindable(false),
+  paths = $bindable([]),
+}: {
+  parentVisible?: boolean;
+  visible?: boolean;
+  paths?: {
+    tagName: string | null;
+    repo: string | null;
+    installPath: string;
+    visible: boolean;
+    isDependency: boolean;
+  }[];
+} = $props();
+
+let selectedBotpackType = $state("official");
+let customRepo = $state("");
+let installPath = $state("");
+
+async function setDefaultPath() {
+  const defaultPath = await App.GetDefaultPath();
+  installPath = `${defaultPath}/RLBotPack`;
+}
```

Comment:
is `App.GetDefaultPath()` next to the binary? In that case, it should be something like ``${defaultPath}/../bots/RLBotPack`` on Windows.

The install directory is currently:
```
%localappdata%/RLBot5/
    bin/
        launcher.exe
        rlbotgui.exe
        RLBotServer.exe
    bots/
        (bot pack here please)
    icon.ico
    uninstall.exe
```